.upgrade-config.yml

```diff
@@ -0,0 +1,6 @@
+# WARNING: This file is autogenerated - changes will be overwritten when regenerated by https://github.com/pulumi/ci-mgmt
+
+---
+upstream-provider-name: terraform-provider-awsx
```

Comment:
Let's have a follow-up work item to remove referencing `terraform`. This could be confusing for users since awsx is not a bridged provider.

Reply:
Good catch, looking.